src/participant/prompts/exportToPlayground.ts

```diff
@@ -0,0 +1,32 @@
+import * as vscode from 'vscode';
+
+export class EportToPlaygroundPrompt {
+  static getAssistantPrompt(): vscode.LanguageModelChatMessage {
+    const prompt = `You are a MongoDB expert.
+Your task is to help the user build MongoDB queries and aggregation pipelines that perform their task.
+You achieve this by converting user's code written in any proggramming language to the MongoDB Shell syntax.
```

Comment:
How do you feel about
```suggestion
You convert user's code written in any programming language to the MongoDB Shell syntax.
```